app/navigation/ManufacturingNavigator.scala

```diff
@@ -44,14 +44,14 @@ class ManufacturingNavigator @Inject()() extends Navigator {
 
   protected val checkRouteMap: PartialFunction[Page, UserAnswers => Call] = {
     case TypeOfManufacturingPage => typeOfManufacturing(CheckMode)
+    case AluminiumApprenticePage => aluminiumApprentice(CheckMode)
     case AluminiumOccupationList1Page => aluminiumOccupationList1(CheckMode)
     case AluminiumOccupationList2Page => aluminiumOccupationList2(CheckMode)
-    case AluminiumOccupationList3Page => aluminiumOccupationList3(CheckMode)
-    case AluminiumApprenticePage => _ => controllers.routes.EmployerContributionController.onPageLoad(CheckMode)
-    case IronApprenticePage => _ => controllers.routes.EmployerContributionController.onPageLoad(CheckMode)
+    case AluminiumOccupationList3Page => _ => controllers.routes.EmployerContributionController.onPageLoad(CheckMode)
+    case IronApprenticePage => _ => routes.IronMiningController.onPageLoad(CheckMode)
```

Comment:
i think if apprentice true it should go to employer contribution, then iron mining if false?